### PR TITLE
Add vokoscreenNG

### PR DESCRIPTION
**Pull request by Mailaender**
### Please confirm your submission meets all the criteria

- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)* The software has support for grabbing from webcams.
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** https://github.com/vkohaupt/vokoscreenNG/issues/21 https://github.com/vkohaupt/vokoscreenNG/issues/58
- [X] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)* https://github.com/vkohaupt/vokoscreenNG/pull/94 https://github.com/vkohaupt/vokoscreenNG/pull/126 https://github.com/vkohaupt/vokoscreenNG/pull/165 https://github.com/vkohaupt/vokoscreenNG/issues/93

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: com.github.vkohaupt.vokoscreenNG.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+command: vokoscreenNG
+rename-desktop-file: vokoscreenNG.desktop
+rename-appdata-file: vokoscreenNG.appdata.xml
+rename-icon: vokoscreenNG
+finish-args:
+  - --device=all
+  - --device=dri
```
Comment (hfiguiere) [inline com.github.vkohaupt.vokoscreenNG.yml:11]:
```suggestion
  - --device=all
```

dri is include in all.

File: vokoscreenNG.appdata.xml [hunk 2]
```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.github.vkohaupt.vokoscreenNG</id>
+  <name>vokoscreenNG</name>
+  <summary>Easy to use screencast creator</summary>
+  <metadata_license>MIT</metadata_license>
```
Comment (hfiguiere) [inline vokoscreenNG.appdata.xml:6]: `CC0-1.0` is prefered.

**Comment by hfiguiere** [comment]
bot, build com.github.vkohaupt.vokoscreenNG

**Review by hfiguiere — CHANGES_REQUESTED** [review]

**Comment by barthalion** [comment]
bot, build com.github.vkohaupt.vokoscreenNG

**Pull request closed without merging**